cookbooks/aws-parallelcluster-config/recipes/update_shared_storages.rb

```diff
@@ -29,9 +29,9 @@
     # get raid to mount
     node.default['cluster']['mount_raid_shared_dir'], node.default['cluster']['mount_raid_type'], node.default['cluster']['mount_raid_vol_array'] = get_raid(MOUNT_ACTION)
     # get efs to unmount
-    node.default['cluster']['unmount_efs_shared_dir_array'], node.default['cluster']['unmount_efs_fs_id_array'] = get_efs(UNMOUNT_ACTION)
+    node.default['cluster']['unmount_efs_shared_dir_array'], node.default['cluster']['unmount_efs_fs_id_array'], = get_efs(UNMOUNT_ACTION)
```

Comment:
Is there an extra "," here?

Reply:
This is the recommended way when omitting values from parallel assignment
```
a,b, = 1,2,3,4
puts a
#1
puts b
#2
```